motoman_driver/MotoPlus/MotionServer.c

```diff
@@ -770,35 +775,16 @@ BOOL Ros_MotionServer_ServoPower(Controller* controller, int servoOnOff)
 	MP_SERVO_POWER_SEND_DATA sServoData;
 	MP_STD_RSP_DATA stdRespData;
 	int ret;
+	STATUS status;
 	
 	if (servoOnOff == OFF)
 		Ros_MotionServer_StopMotion(controller);
 
-	if (servoOnOff == ON && Ros_Controller_IsEcoMode(controller) == TRUE)
+	if (servoOnOff == ON)

```

Comment:
(perhaps just for my understanding): is there a reason `Ros_Controller_IsServoOn(controller)` is not used here?